src/components/add-book.js

```diff
@@ -1,10 +1,33 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/books/books';
 
-const AddBook = () => (
-  <form>
-    <input type="text" placeholder="Book Title" />
-    <input type="select" placeholder="Category" />
-    <input type="button" value="Add Book" />
-  </form>
-);
+const AddBook = () => {
```

Comment:
- [ ] The form responsible for adding a book should be a controlled component. The component should contain: - A local state with `title`, `author` - they should have their inputs that updates the local state every time they update (`onChange`) then dispatches the `addBook` action with the object containing a new id as its argument

Reply:
You mean, have someting like:

Reply:
This issue is well resolved! ✔️